user-service/controller/user-controller.js

```diff
@@ -25,6 +27,20 @@ export async function createUser(req, res) {
     }
 }
 
+export async function deleteUser(req, res) {
+    try {
+        const { username } = req.params;
+        const resp = _deleteUser(username);
+        if (resp) {
+            return res.status(200).json({message: `User successfully removed!`}); 
+        } else {
+            return res.status(404).json({message: 'User not found!'});
+        }

```

Comment:
Currently if resp is an err as seen in the user-orm.js, it is uncertain how this if else block would respond. Neither responses would be appropriate though.

Reply:
ok will made the change to catch the error in resp.